Application/Source/Components/CreateTrip/CreateTripRootView.ui.xml

```diff
@@ -40,4 +40,11 @@
         layout:below="begin offset(35)"
         layout:bottom="super offset(-30)"
         layout:height=":gt 60" />
+
+    <Label
+        field="placeholderLabel"
+        layout:left="comment.left"
```

Comment:
We're using `layout:leading` (instead of `layout:left`) and `layout:trailing` (instead of `layout:right`) in Brightify. Please consider using them instead for consistency.

Another thing, that you may not have heard about, by default the edge that is being layouted (for example `top` here) is moved to the `top` of the other component without you explicitly putting it there. So `layout:top="comment.top"` is equivalent to `layout:top="comment"`. It's not that much of a difference but it makes the code cleaner.